From 82fcdbb86f00963141360e9e7a7771397be58f45 Mon Sep 17 00:00:00 2001
Message-Id: <82fcdbb86f00963141360e9e7a7771397be58f45.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Deepak M <m.deepak@intel.com>
Date: Wed, 24 Sep 2014 09:02:33 +0530
Subject: [PATCH 280/312] FOR_UPSTREAM [VPG]: drm/i915: Correction to VBT
 parsing code comment

we are not skipping the panel id and sequence size instead
we are skipping the sequence byte and size of operation fields.

Issue: IRDA-821
Change-Id: I3295bc5682eb7aa9967f5a5ee159d11afd156d60
Signed-off-by: Deepak M <m.deepak@intel.com>
---
 drivers/gpu/drm/i915/intel_bios.c |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_bios.c b/drivers/gpu/drm/i915/intel_bios.c
index 257a074..32c4677 100644
--- a/drivers/gpu/drm/i915/intel_bios.c
+++ b/drivers/gpu/drm/i915/intel_bios.c
@@ -768,7 +768,11 @@ static u8 *goto_next_sequence_v3(u8 *data, int *size)
 	if (--tmp < 0)
 		return NULL;
 
-	/* Skip the panel id and the sequence size */
+	/*
+	 * Skipping the Sequence byte (1 byte) and
+	 * Size of operation(4 bytes) to
+	 * goto the first element
+	 */
 	data = data + 5;
 	while (*data != 0) {
 		u8 element_type = *data++;
-- 
1.7.9.5

